queryserver-orchestrator/pom.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>phoenix-queryserver</artifactId>
+        <groupId>org.apache.phoenix</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>queryserver-orchestrator</artifactId>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-release-plugin</artifactId>
```

Comment:
We don't usually specify release plugin version in the pom AFAIK. @joshelser Any ideas on this one?

Reply:
The parent pom should contain pluginManagement/dependencyManagement sections which define the versions of plugins/dependencies used in the modules. Be aware that the ASF parent pom also defines plugins, so you often do not need to set one by hand.

Reply:
done

Reply:
Does not look done.